CONTRIBUTING.md

```diff
@@ -0,0 +1,214 @@
+# Contribution Guide 
+
+There are many ways to be an open source contributor, and we're here to help you on your way! You may:
+
+* Propose ideas in our 
+  [discussion forums](https://forums.tbd.website)
+* Raise an issue or feature request in our [issue tracker](LINK_HERE)  ___***FIX LINK AND REMOVE THIS NOTICE***___
+* Help another contributor with one of their questions, or a code review
+* Suggest improvements to our Getting Started documentation by supplying a Pull Request
+* Evangelize our work together in conferences, podcasts, and social media spaces.
+
+This guide is for you.
+
+## Development Prerequisites
+
```

Comment:
Can remove this or we can take it in a follow up

Reply:
Yep you can remove it for now; recommend throwing an issue in there to follow up w/ contribution guide